go.mod

```diff
@@ -24,6 +24,7 @@ require (
 	github.com/jcmturner/gofork v1.0.0 // indirect
 	github.com/jpillora/backoff v0.0.0-20180909062703-3050d21c67d7
 	github.com/kisielk/og-rek v1.0.0
+	github.com/lestrrat-go/strftime v1.0.1
```

Comment:
Why using strftime over time.Format ?

Reply:
strftime provides a simple way to use week number to do a weekly index ("%U").